docs/source/contents/servers/index.md

```diff
@@ -22,6 +22,20 @@ There is a default capabilities for each server as follows:
   :end-before: SELDON_OVERCOMMIT_PERCENTAGE
   ```
 
+## Extra Capabilities
+Servers can be set up with the `extraCapabilities` field to indicate custom configurations (e.g. Python dependencies). For instance:
+
+```{literalinclude} ../../../../samples/servers/mlserver-extra-capabilities.yaml
+:language: yaml
+```
+Note that `serverConfig: mlserver` will provide default capabilities for MLServer as shown above, and the values specified in `extraCapabilities` are appended to them to create a single list of capabilities.
```

Comment:
```suggestion
Note that `serverConfig: mlserver` will provide default capabilities for MLServer as shown above, and the values specified in `extraCapabilities` are appended to them to create a single list of capabilities for this `Server`.
```

Reply:
```suggestion
This server, `mlserver-extra`, inherits a default set of capabilities via `serverConfig: mlserver`.
These defaults are discussed above.
The `extraCapabilities` are appended to these to create a single list of capabilities for this server.
```
Alternative suggestion that perhaps read better?

Reply:
Yes the second suggestion reads better. I added a new commit suggestion below myself as I couldn't apply it to the outdated file.